### ISSUE TITLE
error while running enable_internet_forwarding.sh script

### ISSUE THREAD

**Issue by ketasic** [lucascouto/krackattack-all-zero-tk-key#4]
HI 
when running the enable_internet_forwarding.sh with the command

#sudo ./enable_internet_forwarding.sh

it gives me an error
Command line is not complete. Try option "help"

can you please help me as to what i am doing worng??
thanks

**Comment by sergeyzapor** [comment]
Try:
apt install dnsmasq

**Comment by ketasic** [comment]
Hi.
Sorry for replying late. 
Thanks for replying. 
Yes I installed dnsmasq and all other required packages as required in the Readme file. 
However it gives the same error
"Command line not complete. Try option Help"
Hoping for a reply thanks

**Comment by sergeyzapor** [comment]
I think you dont need to run sudo ./enable_internet_forwarding.sh to execute this script.
I had the same issue in dnsmasq_log. I apt installed dnsmasq and all worked fine

**Comment by r3dw0lf** [comment]
You don't need to run the script, but if you want you need to give all it needs to operate,
which is : 1. internet facing interface 2. Rogue Access point Interface.

**Comment by ketasic** [comment]
Yes you're right. It needs a rogue access point. I checked the dns log files and I think that's where the problem is though I haven't been able to solve it.
I do have dnsmasq installed.

Get Outlook for Android<https://aka.ms/ghei36>
